app/src/pages/inside/projectSettingsPageContainer/content/integrations/integrationsList/integrationInfo/integrationInfo.jsx

```diff
@@ -188,27 +225,16 @@ export const IntegrationInfo = (props) => {
     );
   };
 
+  const getEditAuthConfig = () => ({
```

Comment:
You can wrap it to useMemo which return value or at least just create regular object. Why we should call this function on each re-render?